README.md

```diff
@@ -1,35 +1,60 @@
 
+The aim of the frontend is to feel just as hackable as Pulsar does, all while promoting usage of Pulsar and it's features, while trying to put a more modern spin on how this information is displayed than the `atom.io/packages` page previously did.
+
+### Themes
+
+The Pulsar Package Frontend provides several themes to find the style that suites you:
+  * Atom.io Theme - The default theme which is based off the [Original Atom Package Browser](https://atom.io/packages)
+  * GitHub Dark - A theme based off the GitHub Dark Theme.
+  * Dracula - A theme based off the versatile [Dracula](https://draculatheme.com/) theme🧛
+  * One-Dark - A theme based after the built in Pulsar UI Theme One-Dark.
+
+If there isn't a theme yet that fits you, creating a new theme is welcome, encouraged, and easy.
+
+First create the theme itself, by laying out the theme variables in `./src/site.css` the top of this file should have every other supported theme listed along with the variables that can and need to be modified for your theme.
+
+Once the theme is created, ensure to add it as an option in `./public/site.js` within the function `changeTheme` add a new check in the switch statement. Setting the body to have your theme as an attribute value, and saving the new theme option into the users local storage.
+
+Lastly we need to ensure users are able to choose this theme when they'd like to. Within `./ejs-views/partials/header.ejs` add your new theme as a button towards the bottom of the page, along with all the other theme buttons. Ensure your theme name is passed to the `changeTheme` function.
+
+Now with your theme created double check that the same name is used *exactly* in each location:
+  * The EJS Template
+  * The Client Side JavaScript
+  * The Client Side CSS
+
+Now it's good form to submit your PR pre-fixed with `[THEME]` so we know what exactly to check for.
+
+If you do decide to create your own theme thanks a ton for contributing!
+
+### Data Management
+
+One of the largest tasks of the frontend website is to translate the data returned by the Package Registry API into a properly usable and standardized format for display. The reason for this is there are many variations that can be found when users declare their `package.json`.
+
+If you'd like to read about some of the challenges of normalizing the data across so many different packages:
   * [Repository URLs](/docs/repository-urls.md)
   * [Authors](/docs/authors.md)
   * [Links](/docs/links.md)
 
-Additionally for anyone crafting a direct download link to our Pulsar Binaries from Cirrus CI:
-  * [Download Link Table](/docs/download_links.md)
+### Load Times
 
----
+When browsing the Package Frontend, if you'd like to investigate what's caused the page load to take a long time there's a built in utility to assist with this. While it's unable to show every aspect of the page load it still can be a useful tool to point fault to, and determine where we need to improve.
```

Comment:
```suggestion
When browsing the Package Frontend, if you find that the page load is taking a long time and you would like to investigate the cause, there's a built in utility to assist with this. While it's unable to show every aspect of the page load it can still be a useful tool to point fault to, and determine where we need to improve.
```